frontend/src/components/BackButton.jsx

```diff
@@ -0,0 +1,30 @@
+import { Link } from "react-router-dom";
+
+const BackButton = () => {
```

Comment:
Could we name this better?

`BackButton` is ambiguous and potentially even misleading.
`BackToScenarioSelection` or similar would be more appropriate given what this does. (Redirects to root)